### ISSUE TITLE
Exclude Preview Token for Stories backup

### ISSUE THREAD

**Issue by ralph-kudo** [webflorist/storyblok-backup#3]
For the stories/***.json. This is more of a request, is there a way we can exclude preview_token? I am using this backup hourly and committing the changes on a repo, and most of the diff are just this preview_token prop. 

```
    "preview_token": {
        "token": "f095bbc0454dd******a40633e23f4a83a",
        "timestamp": "1719450145"
    },
    
```

Let me know what you think about it. 

Thanks!

**Comment by codeflorist** [comment]
hey @ralph-burstsms,

i think, that's definitely a good idea. backups should be incremental in a meaningful way.

i'm thinking about removing the `preview_token` by default without even making it an option, since i can't really think of any case, where it is useful. afaik it's only needed for the bridge when editing in the visual editor and i doubt anyone would use this backup package for anything like that. and it's not required for a restore call.

**Comment by codeflorist** [comment]
I've just released v0.1.0, that includes this change. Beware, that i've also added prettifying the JSON-files, resulting in many diffs.
